psutil/tests/test_windows.py

```diff
@@ -156,6 +156,31 @@ def test_total_phymem(self):
         self.assertEqual(int(w.TotalPhysicalMemory),
                          psutil.virtual_memory().total)
 
+    def test_total_swapmem(self):
+        if (psutil.swap_memory().total > 0):
+            w = wmi.WMI().Win32_PerfRawData_PerfOS_Memory()[0]
+            self.assertEqual(int(w.CommitLimit)
+                                - psutil.virtual_memory().total,
+                            psutil.swap_memory().total)
+        else:
+            self.assertEqual(0, psutil.swap_memory().total)
+            self.assertEqual(0, psutil.swap_memory().free)
+            self.assertEqual(0, psutil.swap_memory().used)
```

Comment:
Why compare against WMI only when psutil returns a value != 0? When psutil return 0, what is returned by WMI?

Reply:
I actually previously had another assertion in there including the CommitTotal that would have resulted in a divide by zero. I deleted it so the conditional is no longer required.   I removed it when I did the percent calculation later.

Will update.